packages/squiggle-lang/src/js/index.ts

```diff
@@ -1,7 +1,7 @@
-import {runAll} from '../rescript/ProgramEvaluator.gen';
-import type { Inputs_SamplingInputs_t as SamplingInputs } from '../rescript/ProgramEvaluator.gen';
+import {runAll} from '../rescript/ProgramEvaluator.res';
```

Comment:
@OAGr @Hazelfire I'm not actually aware of how this works. I did this to get the monorepo-level `yarn run` scripts to terminate with success, but I don't know what else it might screw up.

Reply:
this change got backed out of in a subsequent commit